packages/sdk/README.md

```diff
@@ -238,37 +238,37 @@ dialogs.add(
 ### Configure log
 
 You can set custome log level and logger when using this library.
-The default log level is `info` and SDK will print log information to console.
+Logging is turned off by default, you can turn it on by setting custom log function or log level.
+
+#### Enable log by setting log level
+
+When log level is set, a default logger will also be set if not exist. It prints log information to console.
 
 Set log level using the snippet below:
 
 ```ts
 // Only need the warning and error messages.
 setLogLevel(LogLevel.Warn);
+// Can also set another logger if you want to redirect to Application Insights in Azure Function
+setLogger(context.log);
 ```
 
+#### Enable log by setting log function
```

Comment:
Mention setLogLevel won't take effect (both the log level filter and default logger) when setLogFunction executed?
By the way, I'm curious why the log level filter does not apply to setLogFunction... Maybe it's discussed before but I lost it. Can you share again?

Reply:
Oh I misunderstood it, I'll fix this one. Thanks.